metadata/metadata.go

```diff
@@ -390,7 +390,7 @@ func LoadImportance(m *model.Metadata, importanceFile string) error {
 		for index, v := range m.DataResources[0].Variables {
 			// geocoded variables added after ranking on ingest
 			if index < len(metric) {
-				v.Importance = int(metric[index].Data().(float64)) + 1
+				v.Importance = float64(metric[index].Data().(float64))
```

Comment:
The outer `float64( )` isn't necessary here.  The `.(float64)` is a type assertion and will return the `float64` type or panic.  Panicking probably isn't what we want, so better would be to use the checked form of type assertion:

```go
value, ok := metric[index].Data().(float64)
if !ok {
  // handle error here
}
```
For the error handling, what we've done in more recent code is to count the instances of the particular error, and then log it, or generate a new error from it outside of the processing loop if it makes sense for it to be handled elsewhere.  In the checked case I believe value will get set to 0.0 when it fails, which is probably suitable and coupled with a log.